rust/src/payment_info.rs

```diff
@@ -0,0 +1,31 @@
+use serde::{Serialize, Deserialize};
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Huh I wonder if you could set this as a defulat for the application.

Reply:
yeah no idea, I thought the same, did some google, but couldn't find anywhere, even some people asking the same like here: https://www.reddit.com/r/learnrust/comments/ynpxnq/is_there_a_way_to_make_serde_generate_camelcase/